static/js/AddToSourceSheet.jsx

```diff
@@ -120,11 +120,32 @@ class AddToSourceSheetBox extends Component {
           source[language.slice(0,2)] = selectedWords;
         }
       }
-      let postData = {source: JSON.stringify(source)};
-      if (this.props.note) {
-        postData.note = this.props.note;
+      if (this.validateRefs(source.refs)) {
+        let postData = {source: JSON.stringify(source)};
+        if (this.props.note) {
+          postData.note = this.props.note;
+        }
+        $.post(url, postData, this.confirmAdd);
+      }
+  }
+  validateRefs(refs) {
```

Comment:
@stevekaplan123 I think the name should be more specific to validating that the content doesn't contain an image. This should remove the need for the comment right below (line 132). E.g. `checkContentForImages()`